README.md

```diff
@@ -213,6 +213,218 @@ python3 -m pip install -r requirements.txt
 
 ## Usage
 
+![logo](logo/cropped.png)
```

Comment:
I'm not sure if you meant to, but the entire intro, down to the usage has been edited, but pasted above the original usage instructions, instead of replacing it.

Reply:
Yea i was on the wrong branch and I copy-pasted. seems like it got messed up, but PgBiel fixed it